src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -454,6 +456,20 @@ def get_server_version(configuration):
                                 summary='Unable to fetch kubernetes version')
         logger.warning("Unable to fetch kubernetes version: %s\n", e)
 
```

Comment:
Please add another newline here. There is a consistency of 2 line gaps between functions

Reply:
Removed additional line